arm-network/2016-09-01/swagger/routeFilter.json

```diff
@@ -0,0 +1,849 @@
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Route Filter Rule Resource"
+    },
+    "RouteFilterPropertiesFormat": {
+      "properties": {
+        "rules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RouteFitlerRule"
+          },
+          "description": "Collection of RouteFilterRules contained within a route filter."
+        },
+        "peerings": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "./expressRouteCircuit.json#/definitions/ExpressRouteCircuitPeering"
+          },
+          "description": "A collection of references to express route circuit peerings."
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the resource. Possible values are: 'Updating', 'Deleting', 'Succeeded' and 'Failed'."
+        }
+      },
+      "description": "Route Filter Resource"
+    },
+    "RouteFilter": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RouteFilterPropertiesFormat"
+        },
+        "etag": {
+          "type": "string",
+          "description": "Gets a unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "Route Filter Resource."
+    },
+    "RouteFilterListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RouteFilter"
+          },
+          "description": "Gets a list of route filters in a resource group."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "Response for the ListRouteFilters API service call."
+    },
+    "RouteFilterRuleListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RouteFitlerRule"
+          },
+          "description": "Gets a list of RouteFilterRules in a resource group."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "Response for the ListRouteFilterRules API service call"
+    },
+    "BGPCommunity": {
+      "properties": {
```

Comment:
General comment on the description of properties of BGPCommunity: "The descriptions are not that great. They add no value to the generated client side and API documentation. Can you please provide more context and examples in the description?"